libs/akita/src/__tests__/entityStore.spec.ts

```diff
@@ -349,8 +349,8 @@ describe('EntitiesStore', () => {
       store.add(todo);
       store.add(todo2);
       store.remove();
-      expect(store.entities[1]).toBeUndefined();
-      expect(store.entities[2]).toBeUndefined();
+      expect(store._value().entities[1]).toBeUndefined();
```

Comment:
Why did you change everything to _value()?

Reply:
Had compilation error becahse `store.entities` is private, I saw other suites used `store._value().entities` instead